elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -1,79 +1,140 @@
 
   // properties
-  display: inline-block;
-  font-family: #{pfe-var(font-family--heading)};
-  font-size:   #{pfe-var(font-size)};
-  font-weight: #{pfe-var(font-weight--bold)};
-  line-height: #{pfe-var(line-height)};
+  display:         inline-block;
+  position:        relative;
+  vertical-align:  middle;
+  transition:      padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+
   background-color: #{pfe-local(BackgroundColor)};
   border-color:     #{pfe-local(BorderColor)};
-  position: relative;
-  transition: .3s ease padding; // padding #{pfe-var(animation-timing)};
-  line-height: inherit;
-  vertical-align:  middle;
+
   background-color: #{pfe-local(BackgroundColor)};
   border-radius:    #{pfe-local(BorderRadius)};
   border:           #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor)};
+  padding:          #{pfe-local($cssvar: Padding)};
+
+  ::slotted(button),
+  ::slotted(input) {
+    @include pfe-reset-button;
+  }
+
+  ::slotted(*) {
+    display: inline;
+
+    color:           #{pfe-local(Color, $pfe-cta--Color--fallback)} !important;
+    font-family: #{pfe-local($cssvar: FontFamily, $fallback: #{pfe-var(font-family--heading)})};
+    font-size:   #{pfe-local($cssvar: FontSize, $fallback: #{pfe-var(font-size)})};
+    font-weight: #{pfe-local(FontWeight)};
+    line-height: #{pfe-local($cssvar: LineHeight, $fallback: #{pfe-var(line-height)})};
+    text-decoration: #{pfe-local(TextDecoration)} !important;
+    
+    &::before {  // inner border on priority CTAs
+      content: "";
+
+      display:    block;
+      box-sizing: border-box;
+      position:   absolute;
+      top:    2px;
+      left:   2px;
+      bottom: 2px;
+      right:  2px;
+
+      border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} #{pfe-local($cssvar: BorderColor, $region: inner)};
+      border-radius: 2px;
+      
+      outline: none;
+    }
+  }
+
+  ::slotted(*:not(:disabled)) {
+    cursor: pointer;
+  }
 
   .pfe-cta--arrow {
     display: #{pfe-local($cssvar: arrow--Display, $region: arrow)};
-    padding: pfe-local($cssvar: Padding, $region: arrow);
-    fill: #{pfe-local(Color)};
+    padding: #{pfe-local($cssvar: Padding, $region: arrow)};
+    fill:    #{pfe-local(Color)};
     position: relative;
-    top: .1rem; 
-    width: 13px;
+    top:    .1rem;
+    width:  13px;
     height: 13px;
-    transition: .3s ease padding;
+    transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+  }
+
+  ::slotted(:hover) {
+      color: #{pfe-local(Color--hover)} !important;
+      text-decoration: #{pfe-local(TextDecoration--hover)} !important;
+  }
+
+  ::slotted(:focus) {
+    outline: none !important;
+    color: #{pfe-local(Color--focus)} !important;
+    text-decoration: #{pfe-local(TextDecoration--focus)} !important;
+
+    &::before {
+      border: 1px solid #{pfe-local($cssvar: BorderColor--focus, $region: inner)};
```

Comment:
should this use the var "inner border color focus" since its set on the ::before pseudo element?

Reply:
Yup, that's already what's it's doing.